tests/utils/vram_utils.py

```
@@ -90,28 +90,8 @@ def auto_worker_count(
-    env = os.environ.get("NUM_CPUS")
-    if env:
-        try:
-            n = int(float(env))
-            if n > 0:
-                return n
-        except ValueError:
-            pass
+def _cgroup_cpu_budget() -> int | None:
+    """Return the cgroup CPU quota, or ``None`` when no quota is detectable."""
     try:
         quota_s, period_s = open("/sys/fs/cgroup/cpu.max").read().split()[:2]
```

Ruff (0.16.3)

[warning] Use a context manager for opening files

(SIM115)